code/modules/mob/living/death.dm

```diff
@@ -27,14 +27,13 @@
 	return
 
 /mob/living/dust(just_ash, drop_items, force)
-	death(TRUE)
-
 	if(drop_items)
 		unequip_everything()
 
 	if(buckled)
 		buckled.unbuckle_mob(src, force = TRUE)
 
+	death(TRUE)

```

Comment:
why did you move this

Reply:
The order that things occured in led to the user being killed, putting the plushie on the ground, and then every object on their person would be put on top of the plushie, spamming the squeak sound. 
Moving the kill after the unequip prevents the sound file being played a horrifying amount of times by putting the plushie on top

Reply:
is there a situation where you need to die before dropping items?

Reply:
health analyzer maybe? fugg....

Reply:
you can't just trivially move this

Reply:
I'm still trying to think of a way to fix the sound spam without doing it in a shitty way, but thank you for weighing in on this, Oranges. It won't be included in whatever I get this to.